app/layout.tsx

```diff
@@ -13,7 +22,7 @@ export default function RootLayout(
     posts: React.ReactNode,
     users: React.ReactNode,
 ) {
-    const backgroundColor = clsx('bg-slate-100');
+    const backgroundColor = clsx(poppins.className, 'bg-slate-100');
```

Comment:
maybe rename this to bodyClasses when using different css classes, or just put them directly into body since they don't have much logic, ex.: <body className={clsx(poppins.className, 'bg-slate-100')}>...</body>